packages/e2e-test-utils/README.md

```diff
@@ -258,6 +271,10 @@ Disable auto-accepting any dialogs.
 
 Disables Pre-publish checks.
 
+### disableWelcomeGuide
```

Comment:
For the post editor we have a welcome guide as well and instead of a separate function it's an option in `createNewPost`. Might be good to do something similar or if we keep the separate function have it work in both contexts?

Reply:
The action isn't unique to the Site Editor, and complexity will grow as we introduce more editors.

Let's make it part of the `goToSiteEditor` function and disable it by default.